merfi/iso.py

```diff
@@ -36,7 +39,14 @@ def parse_args(self):
         self.source = util.infer_path(parser.unknown_commands)
         self.check_dependency()
         self.make_iso()
+        self.make_sha256sum()
 
     def make_iso(self):
         cmd = ['genisoimage', '-r', '-o', self.output, self.source]
         util.run(cmd)
+
+    def make_sha256sum(self):
+        with open(self.output, 'rb') as isof:
```

Comment:
It is also a bit safer to use `os.path.join(leading_path, trailing_path)` when dealing with paths (vs. just using `+` ) because it ensures proper handling of '/'.

Extra safe is to use `os.path.abspath(self.output)` to ensure that the path is indeed absolute (has no effect if `self.output` is already absolute.